mail.php

```diff
@@ -0,0 +1,106 @@
+<?php
+	function mailing($motionid)
+	{
+		$motionArray = array($motionid);
+		foreach ($motionArray as $motion)
+		{
+			//Database Connection
+			include_once ('include/db-config.php');
+			$motion=$db_con->prepare ("SELECT * from motions where motion_id = :motionid");
+                        $motion->bindParam(':motionid',$motionid);
+                        $motion->execute();
+			$body="<html>
+					<head>
+						<title>Status of Motion</title>
+					</head>
+					<body>";
+                        while ($row=$motion->fetch(PDO::FETCH_ASSOC))
+                        {
+                                $motionid=$row['motion_id'];
+                                $motionname=$row['motion_name'];
+                                $dateadded=$row['dateadded'];
+                                $motiondesc=$row['motion_description'];
+                                $disposition=$row['motion_disposition'];
+
+                               $body .= "<h1>" . $motionname . "</h1>
+                                	<h2>Date Added:</h2>" . $dateadded . "<br />
+                                	<h2>Motion Text</h2>" .
+                                	$motiondesc .
+                                	"<h2>Disposition:</h2>" .
+                                	$disposition;
+			}//End of while
+
+			$body .= "<br /><br />
+					<h2>Current Votes</h2>
+					<table border=\"1\" width=\"100%\">
+					<tr>
+						<th>User</th>
+						<th>Date</th>
+						<th>Vote</th>
+					</tr>";
+
```

Comment:
Do you have any suggestions of what to review DAOs or just do a Google Search for PHP DAO configuration/setup?